src/main/java/styluscar/functionary/FunctionaryResource.java

```diff
@@ -6,44 +6,46 @@
 
 import javax.inject.Inject;
 import javax.transaction.Transactional;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.awt.*;
 import java.util.List;
 
-@Path("functionary")
+@Produces("application/json")
+@Consumes("application/json")
+@Path("/functionary")
 public class FunctionaryResource {
     @Inject
-    FunctionaryService functionaryService;
+    FunctionaryService service;
 
     @GET
     @Path("/functionarys")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Functionary> getAllFunctionarys() {
         try{
-            return this.functionaryService.findAllFunctionary();
+            List<Functionary> functionarys = this.service.findAllFunctionary();
+            return functionarys;
```

Comment:
Use `return this.functionaryService.findAllFuncionary();`

Reply:
done